### PR TITLE
PP3 CH5&6

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: piety-promoted-v3/modernized/05-chapter-05.adoc [hunk 1]
```diff
@@ -668,11 +668,14 @@ being nearly fifty-four years of age.
 [.asterism]
 '''
 ISAAC SHARPLES.
---A short narrative of ISAAC SHARPLES, late of Hitchin,
+--A short narrative of Isaac Sharples, late of Hitchin,
```
Comment (maijapapaya) [inline piety-promoted-v3/modernized/05-chapter-05.adoc]: Not sure if this should be a heading?

File: piety-promoted-v3/modernized/05-chapter-05.adoc [hunk 2]
```diff
@@ -867,3 +875,5 @@ Thus, our dear and worthy friend, after a long and well-spent life, finished his
 and we doubt not,
 has entered that glorious immortality of rest and peace prepared for the righteous.
 His remains were interred in Friends`' burial-ground at Hitchin the 23rd.
+
+--
```
Comment (maijapapaya) [inline piety-promoted-v3/modernized/05-chapter-05.adoc]: Not sure if the testimony should end here, or the paragraph above.

File: piety-promoted-v3/modernized/05-chapter-05.adoc (hunk 1 above)
Comment (jaredh159) [reply piety-promoted-v3/modernized/05-chapter-05.adoc]: yeah, it's a tough one too.  I think let's move it inside of the embed block and make it a sub-heading.  so there will just be the line 670 above with just the name, then the embed.

File: piety-promoted-v3/modernized/05-chapter-05.adoc (hunk 2 above)
Comment (jaredh159) [reply piety-promoted-v3/modernized/05-chapter-05.adoc]: I think you got it right. 👍

**Comment by jaredh159** [comment]
great, thanks! merging.

**Pull request merged**